

security/dummy.c: In function `security_fixup_ops':
security/dummy.c:907: warning: assignment from incompatible pointer type

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/security/dummy.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN security/dummy.c~bsd-secure-levels-lsm-add-time-hooks-fix security/dummy.c
--- 25/security/dummy.c~bsd-secure-levels-lsm-add-time-hooks-fix	2004-09-30 23:18:11.812977528 -0700
+++ 25-akpm/security/dummy.c	2004-09-30 23:18:11.816976920 -0700
@@ -104,7 +104,7 @@ static int dummy_syslog (int type)
 	return 0;
 }
 
-static int dummy_settime (struct timeval *tv, struct timezone *tz)
+static int dummy_settime(struct timespec *ts, struct timezone *tz)
 {
 	if (!capable(CAP_SYS_TIME))
 		return -EPERM;
_
